### PR TITLE
Add support for nerdctl images.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <smahajan@roblox.com>

### CODE REVIEW

File: images.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		return err
+	}
+
+	return printImages(ctx, clicontext, imageList, cs)
+}
+
+func printImages(ctx context.Context, clicontext *cli.Context, imageList []images.Image, cs content.Store) error {
+	quiet := clicontext.Bool("quiet")
+
+	w := tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
```
Comment (AkihiroSuda) [inline images.go:73]: Nit: stdout should be clicontext.App.Writer

**Comment by shishir-a412ed** [comment]
```
root@vagrant:~/go/src/github.com/nerdctl# nerdctl images
REPOSITORY                  TAG       DIGEST ID       CREATED         SIZE
docker.io/library/alpine    latest    c0e9560cda11    25 hours ago    2.7 MiB
docker.io/library/redis     alpine    b0e84b6b9214    25 hours ago    10.0 MiB
root@vagrant:~/go/src/github.com/nerdctl# nerdctl images --quiet
c0e9560cda11
b0e84b6b9214
```

1. The formatting is slightly tilted in the above output, however, the formatting in the actual command ouput is correct.
2. IIRC in the `docker images` output, `IMAGE ID` is something specific to docker. Please let me know if that is not the case, and containerd also has some notion of `IMAGE ID`. For now, I have replaced imageID with DigestID.

/cc @AkihiroSuda

**Comment by AkihiroSuda** [comment]
I think we can just show digest as "IMAGE ID"

File: images.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	if !quiet {
+		fmt.Fprintln(w, "REPOSITORY\tTAG\tDIGEST ID\tCREATED\tSIZE")
```
Comment (AkihiroSuda) [inline images.go:75]: DIGEST ID -> IMAGE ID

File: images.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	}
+
+	for _, img := range imageList {
+		size, err := img.Size(ctx, cs, platforms.Default())
+		if err != nil {
+			return errors.Wrap(err, "failed to compute image size")
+		}
+
+		repositoryTag := strings.Split(img.Name, ":")
```
Comment (AkihiroSuda) [inline images.go:84]: Can we trim "docker.io/library" and "docker.io/" prefix?

File: images.go (hunk 1 above)
Comment (shishir-a412ed) [reply images.go:73]: Does [`ps tabwriter`](https://github.com/AkihiroSuda/nerdctl/blob/master/ps.go#L70) needs to be fixed too?

Comment (shishir-a412ed) [reply images.go]: Fixed.

File: images.go (hunk 2 above)
Comment (shishir-a412ed) [reply images.go]: Fixed.

File: images.go (hunk 3 above)
Comment (shishir-a412ed) [reply images.go:85]: Fixed.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**